src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -371,7 +372,22 @@ function ChildReconciler(shouldClone, shouldTrackSideEffects) {
     element: ReactElement,
     priority: PriorityLevel,
   ): Fiber {
-    if (current === null || current.type !== element.type) {
+    if (
```

Comment:
It looks like the only difference between this branch and the next is that if it's a Fragment fiber, the `pendingProps` should be the children. Can we merge these two branches and only branch on that assignment?